### ISSUE TITLE
Could you add Group to this httptreemux?

### ISSUE THREAD

**Issue by NateLing** [dimfeld/httptreemux#21]
like this:

```Go
router.Group("/html",func(c XXX){
     c.GET("/a",func)
     c.GET("/b",func)
})
```

**Comment by dimfeld** [comment]
I like the idea. I may not have time to do anything related to this for a month or so, but it sounds pretty simple to do, so I may be able to get it done sooner than that. I'll also be glad to look at a PR.

**Comment by dimfeld** [comment]
Closed by #22
